core-bundle/src/Cron/PruneExpiredDataCron.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Cron;
+
+use Contao\Config;
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\CoreBundle\ServiceAnnotation\CronJob;
+use Doctrine\DBAL\Connection;
+
+class PruneExpiredDataCron
```

Comment:
Shouldn't this be `PurgeExpiredDataCron`? I have googled "prune" and it does not seem correct in this context.

Reply:
In my world, purging means deleting all data. Pruning means some data. We're purging expired data but we don't delete all the data so. I don't really care. Feel free to update it.

Reply:
Will do. I got ~4,800 results for `"prune expired"` and ~31,100 for `"purge expired"`, so I guess the latter is more commonly used.